CHANGELOG.next.asciidoc

```diff
@@ -80,6 +80,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 - in module/windows/perfmon, changed collection method of the second counter value required to create a displayable value {pull}32305[32305]
 - Fix and improve AWS metric period calculation to avoid zero-length intervals {pull}32724[32724]
+- Allow filtering on AWS tags by more than 1 value per key. {pull}32775[32775]
```

Comment:
this should go under [Added -> Metricbeat](https://github.com/elastic/beats/blame/9b3343a8aacd3ad6f9c16b63f7d0971f05f1506c/CHANGELOG.next.asciidoc#L134), line 134 more or less